xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -734,12 +734,15 @@ CGUIFontTTF::Character* CGUIFontTTF::GetCharacter(character_t chr, FT_UInt glyph
   wchar_t letter = (wchar_t)(chr & 0xffff);
   character_t style = (chr & 0x7000000) >> 24;
 
+  if (!glyphIndex)
+    glyphIndex = FT_Get_Char_Index(m_face, letter);
```

Comment:
To move this here is good :+1: and also found on https://github.com/xbmc/xbmc/pull/20552/files#diff-13eb64264879fab58d04f4b2504916173159f92a09b714005363f649ca813cfeR1230 to have better there.